release-tools/KUBERNETES_CSI_OWNERS_ALIASES

```diff
@@ -31,14 +31,7 @@ aliases:
 
 # This documents who previously contributed to Kubernetes-CSI
 # as approver.
-emeritus_approver:
-- lpabon
-- sbezverk
-- vladimirvivien
-
-# This documents who previously contributed to Kubernetes-CSI
-# as reviewer.
-emeritus_reviewer:
+emeritus_reviewers:
```

Comment:
According to https://github.com/kubernetes/community/issues/6334#issuecomment-1011438969, this should be "emeritus_approver".

Also, don't drop vladimirvivien. If there's supposed to be only one section, move him there.

Finally, this change needs to be done in https://github.com/kubernetes-csi/csi-release-tools. From there will copy it automatically into all kubernetes-csi repos.